jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/AppEngineAuthentication.java

```diff
@@ -354,15 +354,14 @@ public void setIdentityService(IdentityService identityService) {
     }
 
     /**
-     * Validate a user identity. Validate that a UserIdentity previously created by a call to
-     * {@link #login(String, Object)} is still valid.
+     * Validate a user identity. Validate that a UserIdentity is still valid.
```

Comment:
1) I think "Validate that a UserIdentity is still valid." should have been enough.
2) I think such doc change should have been applied on `LoginService` (as it does not seem to be specific to this impl).
3) I would normally leave out JavaDoc for methods that I override/implement unless they provide specifics for the implementation. In this case we just throw `UnsupportedException`